docs/swagger/sota-device_registry.yml

```diff
@@ -137,6 +137,18 @@ paths:
         description: the vehicle uuid.
         required: true
         type: string
+      - name: offset
+        description: the offset into the list of packages
+        in: query
+        required: false
+        type: integer
+        format: int64
+      - name: limit
+        description: the maximum amount of packages to return, after the offset
+        in: query
+        required: false
+        type: integer
+        format: int64
```

Comment:
Yes, I spoke to Jon about this. I will go through and add all the PaginatedResponses in a separate PR.